catroid/src/main/java/org/catrobat/catroid/ui/dynamiclistview/UtilDynamicListView.java

```diff
@@ -608,7 +616,9 @@ public void notifyListItemTouchActionUp() {
 	private void touchEventsCancelled() {
 		if (cellIsMobile) {
 			View mobileView = getViewForID(mobileItemId);
-			unsetSpriteMobileState();
+			if (forSpriteList) {
+				unsetSpriteMobileState();
+			}
 			aboveItemId = INVALID_ID;
 			mobileItemId = INVALID_ID;
 			belowItemId = INVALID_ID;
```

Comment:
Thx for the hint, it fixes the crashes for copying, renaming Scenes/Sounds/Looks etc. 👍